### PR TITLE
s390: Work around missing button in disk_partition

### PR DESCRIPTION
**Pull request by Soulofdestiny**

### CODE REVIEW

**Comment by Soulofdestiny** [comment]
@okurz

File: lib/partition_setup.pm [hunk 1]
```diff
@@ -25,7 +25,13 @@ my %role = qw(
 );
 
 sub wipe_existing_partitions {
-    send_key $cmd{expertpartitioner};
+    assert_screen('release-notes-button', 5);
```
Comment (okurz) [inline lib/partition_setup.pm]: don't use timeout below default on assert screen. Why not default?

File: lib/partition_setup.pm [hunk 2]
```diff
@@ -25,7 +25,13 @@ my %role = qw(
...
+    if (match_has_tag('bsc#1054478')) {
+        send_key 'alt-x';
+    }
+    else {
+        send_key $cmd{expertpartitioner};
+    }
```
Comment (okurz) [inline lib/partition_setup.pm]: just personal opinion: How about
`send_key match_has_tag('bsc#1054478') ? 'alt-x' : $cmd{expertpartitioner}` ?

File: lib/partition_setup.pm (hunk 1 above)
Comment (Soulofdestiny) [inline lib/partition_setup.pm]: just adopted from here:
https://github.com/okurz/os-autoinst-distri-opensuse/blob/6e7a501576ec7294284beb9aaebe2f19a2fc4bca/tests/installation/releasenotes.pm#L19

File: lib/partition_setup.pm (hunk 2 above)
Comment (Soulofdestiny) [inline lib/partition_setup.pm]: yeah, agreed

**Comment by Soulofdestiny** [comment]
updated

**Pull request merged**